requirements.txt

```diff
@@ -33,3 +33,4 @@ flake8==3.4.1
 dropbox==8.7.1
 git+https://github.com/jschneier/django-storages.git#egg=django-storages
 requests==2.17.1
+djangorestframework==3.8.2
```

Comment:
You probably can move the view to avoid using DRF (it's only a view right?) Probably using View from Django will work similarly.

Reply:
I don't think so, this is used to bypass the default backend for authorisation on the API view so you can make calls with a token.

Reply:
if you just don't make it a LoginView or avoid using one of our mixins it should bypass the authorization view. Django Views don't have any authorization mechanism by default.